src/OpenSheetMusicDisplay/OSMDOptions.ts

```diff
@@ -0,0 +1,29 @@
+import { DrawingParametersEnum } from "../MusicalScore/Graphical/DrawingParameters";
+
+/** Possible options for the OpenSheetMusicDisplay constructor, none are mandatory. */
+export interface IOSMDOptions {
+    /** Not yet supported. Will always beam automatically. */
+    autoBeam?: boolean;
+    /** Automatically resize score with canvas size. Default is true. */
+    autoResize?: boolean;
+    /** Not yet supported. Will always place stems automatically. */
+    autoStem?: boolean;
+    /** Render Backend, will be SVG if given undefined, SVG or svg, otherwise Canvas. */
+    backend?: string;
+    /** Don't show/load cursor. Will override disableCursor in drawingParameters. */
+    disableCursor?: boolean;
+    /** Parameters like drawing a Leadsheet or (Thumbnail) Preview, disabling Cursor. */
+    drawingParameters?: string | DrawingParametersEnum;
+}
+
+/** Handles [[IOSMDOptions]], e.g. returning default options with OSMDOptionsStandard() */
+export class OSMDOptions {
+    /** Returns the default options for OSMD used if no options are given in the constructor. */
+    public static OSMDOptionsStandard(): IOSMDOptions {
```

Comment:
Can you maybe use @return here so that docs are generated correctly? :)
http://typedoc.org/guides/doccomments/

Reply:
Can do, although e.g. `RegExp.test()` also doesn't have an @return notation.
Are you sure you need @return to get correctly generated docs?

Reply:
Actually I don't.

Reply:
![grafik](https://user-images.githubusercontent.com/6751545/45624682-4ecbce00-ba8b-11e8-9905-5a3ff7bc9431.png)
At least you get a clickable link with a correct section heading which otherwise you don't.

Reply:
Right. Maybe one could do a `[[Link]]` instead. I'll do one or the other.
Can you quickly tell me where to find the docs built with `build:doc`? Can't find them right now.

Reply:
It seems like it gives you a clickable return type without the @return, which is good to know.
![image](https://user-images.githubusercontent.com/33069673/45624934-f9dc8780-ba8b-11e8-8f81-bea1ac7a7928.png)

Reply:
The reason i don't like the @return is that i would need to repeat myself from the description, so if i can describe function and return value in one doc sentence, i like that better.